lib/cloudkeeper/aws/cloud.rb

```diff
@@ -73,25 +73,28 @@ def disk_container(appliance)
       # @param import_id [String] id of import image task
       # @raise [Cloudkeeper::Aws::Errors::BackendError] if polling timed out
       def poll_import_task(import_id)
```

Comment:
I wonder how you're going to handle the exceptions in the method which will be caling this one. You will have to catch both `Cloudkeeper::Aws::Errors::BackendError` and `Cloudkeeper::Aws::Errors::TimeoutError`. And maybe that's ok, maybe you will handle them differently. But I would suggest creating a hierarchy of exceptions so you don't have to if you don't need it. Something like
```
Cloudkeeper::Aws::Errors::BackendError <- Cloudkeeper::Aws::Errors::Backend::ImageImportError
                                       <- Cloudkeeper::Aws::Errors::Backend::TimeoutError
```
This way you can catch only `Cloudkeeper::Aws::Errors::BackendError` if you don't need to distinguish between the errors but you still can if you need to.